crates/zksync/core/Cargo.toml

```diff
@@ -20,6 +20,9 @@ alloy-network.workspace = true
 alloy-sol-types.workspace = true
 alloy-transport.workspace = true
 alloy-zksync.workspace = true
+tokio.workspace = true
+futures.workspace = true
+lazy_static = "1.4"
```

Comment:
we use `lazy_static` now? we don't really need it since it's included in std.. and I also don't see usage